### DISCUSSION TITLE
Using page-flip animations outside of touchscreen mode

### DISCUSSION THREAD

**Discussion by Mosswool** [johnfactotum/foliate#1339] [Q&A] [answered]
I'm using the flatpak version, but desperately want to be able to see the pageflip animations that [I learned](https://github.com/johnfactotum/foliate/commit/51ddfac39c41913c560dd5f19bc74234f29d8300) were added in `51ddfac`.

It seems they were intended for touchscreen users. Is there still a way to see them in the flatpak version on a non-touchscreen device? Thank you in advance.

**Comment by johnfactotum** [comment, marked as answer]
Slide animation is there no matter how you turn the page, unless you turn it off by enabling <i>Reduce animation</i> in <i>Font & Layout Preferences</i>.

If you're talking about gestures, it has both touchscreen and touchpad gestures. You may need Wayland for the latter to work.

**Reply by Mosswool** [reply]
I see, thank you. I mistakenly thought the added feature was a different type of animation than what I had. Thank you for clarifying.
